test/eventbrite-provider.test.ts

```diff
@@ -1,21 +1,38 @@
 /* Copyright © 2021 Seneca Project Contributors, MIT License. */
 
 import * as Fs from 'fs'
+import crypto from 'crypto';
+import dotenv from 'dotenv'
 
 import EventbriteProvider from '../src/eventbrite-provider'
 
+dotenv.config();
 const Seneca = require('seneca')
 const SenecaMsgTest = require('seneca-msg-test')
 const EventbriteProviderMessages = require('./eventbrite-provider.messages').default
 
-const CONFIG: any = {}
+const CONFIG: any = {key : process.env.EVENTBRITE_API_KEY}
```

Comment:
Fine for now, added issue to @seneca/povider: https://github.com/senecajs/seneca-provider/issues/1